### PR TITLE
PUT on /filter-outputs/{id} returns 404 when no auth key is provided

### PR DESCRIPTION
**Pull request by Matt-Guest**
### What

hide authenticated endpoints

### How to review

review the code, run unit tests and acceptance tests

### Who can review

anyone

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,6 +13,8 @@ build:
 	go build -o $(BUILD_ARCH)/$(BIN_DIR)/dp-filter-api cmd/$(MAIN)/main.go
 debug: build
 	HUMAN_LOG=1 go run -race cmd/$(MAIN)/main.go
+acceptance: build
```
Comment (gedge) [inline Makefile]: this is probably a pattern we should use elsewhere (I've been bitten by this, today, on another PR!)

is this documented/used anywhere that you can point to (README should mention it?)?

File: Makefile [hunk 2]
```diff
@@ -13,6 +13,8 @@ build:
...
+	 MONGODB_FILTERS_DATABASE=test HUMAN_LOG=1 go run -race cmd/$(MAIN)/main.go
 test:
 	go test -cover $(shell go list ./... | grep -v /vendor/)
 .PHONY: build debug test
```
Comment (gedge) [inline Makefile]: consider adding `acceptance` here

File: Makefile (hunk 1 above)
Comment (gedge) [inline Makefile]: actually, I just noticed that this should probably not depend on `build` as it `run`s it directly

Comment (Matt-Guest) [reply Makefile]: I removed it, I also did this for `debug` as its the same command

File: Makefile (hunk 2 above)
Comment (Matt-Guest) [reply Makefile]: Done

File: Makefile (hunk 1 above)
Comment (Matt-Guest) [reply Makefile]: I thrown a section in the readme to cover running the acceptance tests

**Pull request merged**